README.md

```
@@ -12,31 +12,59 @@
 - 예매 완료를 확인하는 다이얼로그가 표시되고 배경을 터치해도 사라지지 않아야 한다.
 - 다이얼로그에서 예매 완료를 선택하면 예매 내역 화면으로 이동한다.
 
+사용자는 영화 목록에서 광고를 볼 수 있다.
+- 영화 목록에 영화가 세 번 노출될 때마다 광고가 한 번 노출된다.
+- 영화 목록의 요소는 10,000개까지 추가될 수 있다.
+- 사용자는 인원/날짜 선택 후에 좌석을 고를 수 있다.
+- 좌석을 선택하면 배경색이 바뀌고, 하단에 선택한 좌석 수를 반영한 최종 가격이 표시된다.
+  - 선택된 좌석을 재선택하면 선택이 해제된다.
+
 #### 날짜 및 시간 정책
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)